src/lib.rs

```diff
@@ -111,6 +111,7 @@
 )]
 #![deny(warnings)]
 #![cfg_attr(not(feature = "std"), no_std)]
+#![feature(const_fn)]
```

Comment:
this is not allowed for the stable build. you will have to wrap it in a `cfg_if`